lib/ht_traject/print_holdings_api.rb

```diff
@@ -0,0 +1,60 @@
+require_relative "../services"
+
+module HathiTrust
+  class PrintHoldingsAPI
+
+    def self.connection 
+      @conn ||= Faraday.new( url: ENV["HOLDINGS_API_URL"] ) do |builder|
+        builder.request :json
+        builder.response :json
+        # TODO: Perhaps if it fails we should retry instead of immediately raising an
```

Comment:
@moseshll Any thoughts on behavior here?

Reply:
We have the `faraday-retry` gem (https://github.com/lostisland/faraday-retry) that is currently supported. I'd be meh about a roll-your own but if this works with minimal extra config then I'd say it's worth a go.

Might be better to have retries be a separate ETT coming soon in the pipeline, than clutter up this branch with a nice-ta-have.

Reply:
OK. I can give it a try; if it seems to work without significant issues then I'll leave it for later. If it's clear we need to add retry now or more sophisticated handling to get a full index to complete in testing then I'll add it now.